modules/http4s/test/src/smithy4s/http4s/SimpleRestJsonComplianceSuite.scala

```diff
@@ -95,7 +95,8 @@ object SimpleRestJsonComplianceSuite extends ProtocolComplianceSuite {
       val baseUri = uri"http://localhost"
       val suppliedHost =
         testHost.map(host => Uri.unsafeFromString(s"http://$host"))
-      SimpleRestJsonBuilder(service)
+      SimpleRestJsonBuilder
+        .withRawHttpLabelValues(false)(service)
```

Comment:
I was under the impression that this would default to true to retain the old behavior, but perhaps I'm turned around here 😅

Reply:
this needs to be false to pass the restJson1 compliance tests

Reply:
Oh my bad, I missed this was in tests.